test/lib/core/files.js

```diff
@@ -58,7 +59,7 @@ describe('Files', () => {
 
   describe('dataPath', () => {
     it('should return the data path', () => {
-      const expected = path.join(ospath.data(), 'polkadot-deployer');
+      const expected = strings.removePathSpaces(path.join(ospath.data(), 'polkadot-deployer'));
```

Comment:
Why do we need this here? as i understand it the new function is not called on each of the subject functions, is that correct?

Reply:
I could do definitely something better, let me show you in a sec

Reply:
but still we need it, and yes you are right: it is not called on each of them, but each of them are relying on the same this.config.dataPath = strings.removePathSpaces(ospath.data());